src/main/java/school/lemon/changerequest/java/generics/generator/NumberGenerator.java

```diff
@@ -0,0 +1,13 @@
+package school.lemon.changerequest.java.generics.generator;
+
+import school.lemon.changerequest.java.generics.container.GenericContainer;
+
+import java.util.Random;
+
+public abstract class NumberGenerator {
+
+    protected Random generator = new Random(System.currentTimeMillis());
```

Comment:
Never do such again.
If you need any Random generator — pass it as a constructor argument, or create a `setter` function (constructor is better).